src/patternfly/layouts/Flex/flex.scss

```diff
@@ -311,5 +332,77 @@ $pf-l-flex--variable-map: build-variable-map("pf-l-flex--spacer", $pf-l-flex--sp
       }
     }
   }
+
+
+  // .pf-m-gap-{size}{-on-breakpoint}
+  @each $breakpoint, $breakpoint-value in $pf-l-flex--breakpoint-map {
```

Comment:
If you were to move line 338 - 406 above line 320, the spacer could override gap settings.